tests/commands/test_bump_recipe.py

```diff
@@ -27,9 +28,14 @@ def test_bump_recipe_cli(fs: FakeFilesystem) -> None:
     fs.add_real_directory(get_test_path(), read_only=False)
 
     recipe_file_path = get_test_path() / "simple-recipe.yaml"
+    incremented_recipe_file_path = get_test_path() / "bump_recipe/incremented_by_one.yaml"
 
     result = runner.invoke(bump_recipe.bump_recipe, ["--build-num", str(recipe_file_path)])
-    # TODO: check that the build number has been updated by 1
+
+    parser = load_recipe(recipe_file_path, RecipeParser)
+    incremented_parser = load_recipe(incremented_recipe_file_path, RecipeParser)
+
+    assert parser.render() == incremented_parser.render()
```

Comment:
As a heads-up, the `==` operator is defined to do exactly this: https://github.com/conda-incubator/conda-recipe-manager/blob/main/conda_recipe_manager/parser/recipe_reader.py#L561

Reply:
Got it!